hack/.vendor-helpers.sh

```diff
@@ -70,15 +70,17 @@ clean() {
 		github.com/docker/docker/integration-cli # external tests
 	)
 
-	local dockerPlatforms=( linux/amd64 $(_dockerfile_env DOCKER_CROSSPLATFORMS) )
+	local dockerPlatforms=( linux/amd64 windows/amd64 $(_dockerfile_env DOCKER_CROSSPLATFORMS) )
```

Comment:
@tianon - DOCKER_CROSSPLATFORMS is not set when running ./vendor.sh directly from the hack directory. That's why I added it here.